src/test/java/uk/co/ramp/covid/simulation/ModelTest.java

```diff
@@ -43,7 +43,7 @@ public void testBaseLine() {
             // Check there are deaths, but not too many
             // Near the start of runs this is sometimes untrue, e.g. we may have 8 recoveries and 1 death,
             // so we wait for enough recoveries for this to make sense.
-            if (s.getRecovered() >= 10) {
+            if (s.getRecovered() >= 50) {

```

Comment:
It's possible we just want to check the overall death/recovered ratio near the end, but this lets us see if they randomly shoot up. With the neighbors fixed we had some cases of 24 recoveries/3 deaths which broke this so cranking up the minimal requirement a bit.